org.mapstruct.eclipse/src/org/mapstruct/eclipse/internal/Bindings.java

```diff
@@ -39,25 +44,26 @@ private Bindings() {
      * @param type the type
      * @return the method names declared in the class or a super type of it
      */
-    public static Set<String> findAllMethodNames(ITypeBinding type) {
-        Set<String> result = new HashSet<String>();
+    public static Collection<String> findAllMethodNames(ITypeBinding type) {
```

Comment:
Out of interest, what's the driver for this change? I prefer to return `List` or `Set` as the collection semantics should be apparent to the caller. I realize it's rather philosophical here, just to satisfy my interest :)